recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
 {% set name = "root" %}
-{% set tag_name = "6-28-04" %}
+{% set tag_name = "6-28-08" %}
 {% set version = ".".join(tag_name.split("-")|map("int")|map("string")) %}
-{% set build_number = 2 %}
+{% set build_number = 0 %}
 {% set clang_version = "13.0.1" %}
-{% set clang_patches_version = "root_62804" %}
+{% set clang_patches_version = "root_62808" %}
```

Comment:
I guess you haven't made the PR to conda-forge/clangdev-feedstock to update the patches for this?

Reply:
No, thanks for the remark. I will check which patches are needed for later versions of 6.28 and upload the PR to clangdev